### PR TITLE
Initialize fork for subspace

### PR DESCRIPTION
**Pull request by marc-aurele-besner**
## Initialize fork for subspace

- Change repo for subspace usage
- Change packages/types
- Change packages/adapter
- Change packages/snap
- Change packages/example

### CODE REVIEW

File: packages/snap/src/configuration/index.ts [hunk 1]
```diff
@@ -1,23 +1,23 @@
...
-    case 'westend':
-      console.log('Westend configuration selected');
-      return westendConfiguration;
+    case 'gemini-3g':
+      console.log('Gemini-3g configuration selected');
+      return gemini3gConfiguration;
+    case 'gemini-3f':
+      console.log('Gemini-3f configuration selected');
+      return gemini3fConfiguration;
+    case 'devNet':
```
Comment (0xjeremyfrank) [inline packages/snap/src/configuration/index.ts]:
```suggestion
    case 'devnet':
```

File: packages/snap/src/configuration/index.ts [hunk 2]
```diff
@@ -1,23 +1,23 @@
...
+      console.log('DevNet configuration selected');
+      return devNetConfiguration;
```
Comment (0xjeremyfrank) [inline packages/snap/src/configuration/index.ts]:
```suggestion
      console.log('devnet configuration selected');
      return devnetConfiguration;
```

File: packages/snap/src/configuration/predefined.ts [hunk 3]
```diff
@@ -1,36 +1,36 @@
...
+    symbol: 'tSSC'
   },
-  wsRpcUrl: 'https://westend-rpc.polkadot.io/'
+  wsRpcUrl: 'https://rpc-0.gemini-3f.subspace.network/ws'
 };
 
-export const polkadotConfiguration: SnapConfig = {
-  addressPrefix: 0,
-  networkName: 'polkadot',
+export const devNetConfiguration: SnapConfig = {
```
Comment (0xjeremyfrank) [inline packages/snap/src/configuration/predefined.ts]:
```suggestion
export const devnetConfiguration: SnapConfig = {
```

File: packages/snap/src/configuration/predefined.ts [hunk 4]
```diff
@@ -1,36 +1,36 @@
...
+  addressPrefix: 2254,
+  networkName: 'devNet',
```
Comment (0xjeremyfrank) [inline packages/snap/src/configuration/predefined.ts]:
```suggestion
  networkName: 'devnet',
```

File: packages/snap/src/polkadot/account.ts [hunk 5]
```diff
@@ -28,10 +28,9 @@ export async function getKeyPair(): Promise<KeyringPair> {
 
 const getCoinTypeByNetwork = (network: SnapNetworks): number => {
   switch (network) {
-    case 'kusama':
-    case 'westend':
-      return 434;
-    case 'polkadot':
+    case 'gemini-3g':
+    case 'gemini-3f':
+    case 'devNet':
```
Comment (0xjeremyfrank) [inline packages/snap/src/polkadot/account.ts]:
```suggestion
    case 'devnet':
```

File: packages/types/index.d.ts [hunk 6]
```diff
@@ -124,7 +124,7 @@ export interface UnitConfiguration {
   customViewUrl?: string;
 }
 
-export type SnapNetworks = 'polkadot' | 'kusama' | 'westend';
+export type SnapNetworks = 'gemini-3f' | 'gemini-3g' | 'devNet';
```
Comment (0xjeremyfrank) [inline packages/types/index.d.ts]:
```suggestion
export type SnapNetworks = 'gemini-3f' | 'gemini-3g' | 'devnet';
```

File: packages/example/src/services/format.ts [hunk 7]
```diff
@@ -4,10 +4,10 @@ export function shortAddress(address: string): string {
 
 export function getCurrency(network: string): string {
   switch (network) {
-    case 'kusama':
-      return 'KSM';
-    case 'westend':
-      return 'WND';
+    case 'gemini-3g':
+    case 'gemini-3f':
+    case 'devNet':
```
Comment (0xjeremyfrank) [inline packages/example/src/services/format.ts]:
```suggestion
    case 'devnet':
```

File: packages/example/src/containers/Dashboard/Dashboard.tsx [hunk 8]
```diff
@@ -98,10 +98,10 @@ export const Dashboard = (): React.JSX.Element => {
...
             >
               <InputLabel>Network</InputLabel>
-              <Select defaultValue={'westend'} onChange={handleNetworkChange}>
-                <MenuItem value={'westend'}>Westend</MenuItem>
-                <MenuItem value={'kusama'}>Kusama</MenuItem>
-                <MenuItem value={'polkadot'}>Polkadot</MenuItem>
+              <Select defaultValue={'gemini-3g'} onChange={handleNetworkChange}>
+                <MenuItem value={'gemini-3g'}>Gemini-3G</MenuItem>
+                <MenuItem value={'gemini-3f'}>Gemini-3F</MenuItem>
+                <MenuItem value={'devNet'}>DevNet</MenuItem>
```
Comment (0xjeremyfrank) [inline packages/example/src/containers/Dashboard/Dashboard.tsx]:
```suggestion
                <MenuItem value={'devnet'}>Devnet</MenuItem>
```

File: packages/example/src/@types/index.d.ts [hunk 9]
```diff
@@ -1,8 +1,8 @@
-import type { SnapConfig } from '@chainsafe/metamask-polkadot-types';
+import type { SnapConfig } from '@subspace/metamask-subspace-types';
 
-declare module '@chainsafe/metamask-polkadot-adapter' {
-  export function injectMetamaskPolkadotSnapProvider(
-    network: 'westend' | 'kusama',
+declare module '@subspace/metamask-subspace-adapter' {
+  export function injectMetamaskSubspaceSnapProvider(
+    network: 'gemini-3f' | 'gemini-3g' | 'devNet',
```
Comment (0xjeremyfrank) [inline packages/example/src/@types/index.d.ts]:
```suggestion
    network: 'gemini-3f' | 'gemini-3g' | 'devnet',
```

File: packages/adapter/README.md [hunk 10]
```diff
@@ -36,23 +49,24 @@ function initPolkadotSnap(
...
+```typescript
 SnapConfig {
   networkName: SnapNetworks;
   wsRpcUrl?: string;
   addressPrefix?: number;
   unit?: UnitConfiguration;
 }
 
-SnapNetworks = "polkadot" | "kusama" | "westend";
+SnapNetworks = "gemini-3f" | "gemini-3g" | "devNet";
```
Comment (0xjeremyfrank) [inline packages/adapter/README.md]:
```suggestion
SnapNetworks = "gemini-3f" | "gemini-3g" | "devnet";
```

File: packages/snap/src/configuration/index.ts [hunk 11]
```diff
@@ -1,23 +1,23 @@
...
+import type { SnapConfig } from '@subspace/metamask-subspace-types';
 import { getMetamaskState } from '../rpc/getMetamaskState';
 import {
   defaultConfiguration,
-  kusamaConfiguration,
-  polkadotConfiguration,
-  westendConfiguration
+  gemini3gConfiguration,
+  gemini3fConfiguration,
+  devNetConfiguration
```
Comment (0xjeremyfrank) [inline packages/snap/src/configuration/index.ts]:
```suggestion
  devnetConfiguration
```

**Review by 0xjeremyfrank — COMMENTED** [review]
`devNet` should be `devnet` for consistency.

**Comment by marc-aurele-besner** [comment]
> `devNet` should be `devnet` for consistency.

Thank you for these changes 👍

**Review by 0xjeremyfrank — APPROVED** [review]
lgtm

**Pull request merged**